.theme/app/composables/useGitHub.js

```diff
@@ -1,3 +1,5 @@
+import { useRuntimeConfig } from "#app";
+
```

Comment:
do we need this import? i think autoimport provides `useRuntimeConfig` as well.

Reply:
Sometimes Eslint would get angry. I thought that if i got angry, it might cause problems in the actions too

Reply:
ok :raised_hands: 

but it doesn't get "angry" in other files? i've seen runtime config usage without this import in `.vue` files... :thinking: